src/style.css

```diff
@@ -1 +1,37 @@
 /* styles go here */
+
+.per_info {
+  margin-bottom: 20px;
+}
+
+.reg_info {
+  margin-bottom: 20px;
+}
+
+.about_you_info {
+  margin-bottom: 20px;
+}
+
+.additional_info {
+  margin-bottom: 20px;
+}
```

Comment:
don't repeat yourself. you use the same style for a lot of classes
you could use one common class or apply styles using coma
```
.style1, style2, style 3 {
  margin-bottom: 20px;
}
```